spec/truffle/interop/special_forms_spec.rb

```diff
@@ -70,6 +70,34 @@
     pa.log.should include([:polyglot_read_array_element, 0])
   end
 
+  it description['.at(index)', :at, (:index)] do
```

Comment:
The second argument should be the primary interop message that is used, in this case that's `readArrayElement`.
The third argument should be an Array like above.
Same below.